suite-common/toast-notifications/src/notificationsActions.ts

```diff
@@ -48,6 +54,13 @@ export const addToastOnce = createActionWithExtraDeps(
     },
 );
 
+export const removeToastByType = createAction(
+    `${ACTION_PREFIX}/removeToastByType`,
+    (payload: { toastType: ToastNotification['type'] }) => ({
```

Comment:
nit: consider creating `ToastNotificationType` type (sounds crazy in this sentence, but same thing as `NetworkSymbol`

Reply:
fixed ce38fec3e